proto/cometbft/types/v1/params.proto

```diff
@@ -58,7 +58,7 @@ message ValidatorParams {
   option (gogoproto.populate) = true;
   option (gogoproto.equal)    = true;
 
-  repeated string pub_key_types = 1;
+string pub_key_type = 1;
```

Comment:
```suggestion
  string pub_key_type = 1;
```

formatting seems off

Reply:
This breaks the field type without changing the field number.